kalliope/core/ConfigurationManager/SettingLoader.py

```diff
@@ -49,11 +51,11 @@ class SettingLoader(object):
     __metaclass__ = Singleton
 
     def __init__(self, file_path=None):
-        logger.debug("Loading settings with file path: %s" % file_path)
         self.file_path = file_path
         if self.file_path is None:
-            # use default file if not provided
-            self.file_path = FILE_NAME
+            self.file_path = self._get_settings_file_path()
```

Comment:
as the file_path  is none here, why pass it to the next method? it will be null anyway.